src/main/kotlin/no/nav/medlemskap/domene/Oppholdstillatelse.kt

```diff
@@ -52,7 +52,12 @@ data class Arbeidsadgang(
 data class OppholdstillatelsePaSammeVilkar(
     val periode: Periode?,
     val type: OppholdstillaelsePaSammeVilkarType?,
-    val harTillatelse: Boolean?
+    val harTillatelse: Boolean?,
+    val oppholdPaSammeVilkar: OppholdPaSammeVilkar?
```

Comment:
Trenger ikke egen klasse for den

Reply:
Hvilken informasjon det flagget oss egentlig? Kanskje det er mulig å gi et bedre navn

Reply:
Undergruppen Opphold på samme vilkår omfatter personer med utløpt midlertidig oppholdstillatelse, som har søkt fornyet oppholdstillatelse eller oppholdstillatelse på nytt grunnlag innen fristen[1] og har derfor i de fleste tilfeller rett til opphold på samme vilkår i medhold av tidligere tillatelse fram til søknaden om oppholdstillatelse er behandlet.

Kanskje utloptMidlertidigOppholdMedFornyetSoknad ?